ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc

```diff
@@ -26,10 +26,10 @@
 
 #include "ngraph/event_tracing.hpp"
 #include "ngraph/runtime/backend.hpp"
+#include "ngraph_bridge/ngraph_freshness_tracker.h"
+#include "ngraph_bridge/ngraph_timer.h"
+#include "ngraph_bridge/ngraph_utils.h"
 #include "ngraph_catalog.h"
```

Comment:
Missed this one?

Reply:
`ngraph_catalog.h` and `ngraph_catalog.h` are in the same folder

Reply:
Yes - need to add complete paths even if they are in the same folder. Please see examples from TensorFlow source code.